subsys/bluetooth/controller/ll_sw/ull_llcp_local.c

```diff
@@ -197,6 +197,31 @@ struct proc_ctx *llcp_lr_peek_proc(struct ll_conn *conn, uint8_t proc)
 	return ctx;
 }
 
+#if defined(CONFIG_BT_CTLR_CENTRAL_ISO)
+struct proc_ctx *llcp_lr_peek_cc(struct ll_conn *conn, uint16_t cc_handle)
+{
+	/* This function is called from both Thread and Mayfly (ISR),
+	 * make sure only a single context have access at a time.
+	 */
+
+	struct proc_ctx *ctx, *tmp;
+
+	bool key = shared_data_access_lock();
```

Comment:
`shared_data_access_lock` has a return type of `uint32_t`, but is always returning a boolean value. Is that intentional? 

And is `key` an appropriate name for a boolean value?

Reply:
Fair point since it an existing issue